### PR TITLE
Links optionally CDN

### PR DESCRIPTION
**Pull request by bkase**
When you render with prod, we try to load assets from the CDN:

Screen Shot 2019-04-02 at 3 55 45 PM

I did an exhaustivity search by visiting the pages and making sure the assets didn't load anywhere. I'll go ahead and upload the CDN assets for real after this lands.

### CODE REVIEW

File: frontend/website/src/filter.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env node
+
+// Pandoc filter to convert all text to uppercase
```
Comment (Schmavery) [inline frontend/website/src/filter.js]: This comment probably isn't true anymore

File: frontend/website/src/Head.re [hunk 2]
```diff
@@ -57,19 +65,19 @@ let make = (~extra, ~filename, _children) => {
       <link
         rel="stylesheet"
         type_="text/css"
-        href="/static/css/normalize.css"
+        href={Links.Cdn.url("/static/css/normalize.css")}
       />
       <link rel="stylesheet" type_="text/css" href={filename ++ ".css"} />
```
Comment (Schmavery) [inline frontend/website/src/Head.re]: Which css are we putting on the CDN and which are we not?

Comment (bkase) [reply frontend/website/src/Head.re]: All of it if we can -- I see I missed the "filename.css" ones. It's a bit hard to do the ones that are siblings of the jobs pages and such so maybe we can leave those out. Do you think that's okay?

font.css is super important to be CDN-ed for example.

Comment (Schmavery) [reply frontend/website/src/Head.re]: I think it makes sense for the sibling css to live with the html and for the font.css to be special-cased, yeah

File: frontend/website/src/Home.re [hunk 3]
```diff
@@ -1,4 +1,9 @@
-let extraHeaders = <link rel="stylesheet" type_="text/css" href="index.css" />;
+let extraHeaders =
+  <link
+    rel="stylesheet"
+    type_="text/css"
+    href={Links.Cdn.url("/index.css")}
+  />;
```
Comment (Schmavery) [inline frontend/website/src/Home.re]: I think this extra headers might not actually be necessary

**Pull request merged**